mp/src/game/server/momentum/mom_triggers.h

```diff
@@ -246,13 +246,19 @@ class CTriggerTimerStop : public CTriggerZone
 public:
     DECLARE_CLASS(CTriggerTimerStop, CTriggerZone);
     DECLARE_NETWORKCLASS();
+    DECLARE_DATADESC();
 
     virtual void Spawn() OVERRIDE;
 
     // always send to all clients
     virtual int UpdateTransmitState() OVERRIDE { return SetTransmitState(FL_EDICT_ALWAYS); }
 
     int GetZoneType() OVERRIDE;
+
+    const bool GetCancelBool() { return m_bCancel; }
+
+private:
+    bool m_bCancel;
```

Comment:
You should initialize this to false in an added constructor to this trigger class.